package.json

```diff
@@ -1,8 +1,19 @@
 {
   "name": "chcli",
-  "version": "1.0.0",
+  "version": "1.0.2",
   "description": "A command line utility to make create clubhouse stories easier",
-  "main": "index.js",
+  "main": "dist/src/index.js",
+  "publishConfig": {
+    "registry": "http://npm.adrise.tv"
```

Comment:
should we do this in a public repo? i image we should publish this package to official npm registry.

Reply:
Yes, eventually I will